packages/schematics/schematics/package.json

```diff
@@ -11,6 +11,7 @@
   "schematics": "./collection.json",
   "dependencies": {
     "@angular-devkit/core": "0.0.0",
-    "@angular-devkit/schematics": "0.0.0"
+    "@angular-devkit/schematics": "0.0.0",
+    "rxjs": "6.3.2"
```

Comment:
Why does this need to be added here? AFAIK schematics/schematics do not directly depend on `rxjs`